config/vsphere-performance.metrics

```diff
@@ -1,10 +1,9 @@
-# The list specifies the performance counters grouped by entity type and level.
-# The current list contains a subset of the performance counter metrics available, moreover they could
-# vary depending on the vCenter version and configuration.
-# As a best practice, only metrics that are needed should be collected since the number of metrics
-# and their level affects the vCenter performance. 
-# For more information regarding performance metrics, please check out the official vmWare documentation:
-# https://docs.vmware.com/en/VMware-vSphere/6.7/vsphere-esxi-vcenter-server-67-monitoring-performance-guide.pdf
+# The current list contains counters most of the performance counter metrics but they could
```

Comment:
I believe that these are old, the version before was better

Reply:
yes i didn't realize that you updated the content i just reverted